tasks/selinux.yml

```diff
@@ -0,0 +1,75 @@
+---
+# SELinux tasks for the duffy role
+
+- name: Install packages needed to build SELinux modules
+  package:
+    name:
+      - checkpolicy
+      - policycoreutils
+    state: installed
+  tags:
+    - pkgs
+
+- name: Install directory for custom SELinux module
+  file:
+    path: "{{ duffy_virtualenv }}/selinux"
+    state: directory
+  tags:
+    - files
+
+- name: Install SELinux module for Duffy to talk to Redis and PostgreSQL
+  template:
+    src: local-duffy.te.j2
+    dest: "{{ duffy_virtualenv }}/selinux/local-duffy.te"
+  register: local_duffy_te_file
+  tags:
+    - files
+
+- name: Compile SELinux module for Duffy to talk to Redis and PostgreSQL
+  command:
+    chdir: "{{ duffy_virtualenv }}/selinux"
+    cmd: checkmodule -M -m -o local-duffy.mod local-duffy.te
```

Comment:
Ideally we'd not have to compile locally on the node any selinux policy but rather distribute it from ansible, but let's just verify why you think we need one, and review this later

Reply:
As we discussed on the call, let's try to make it so the services run unconfined.